pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1750,3 +1756,22 @@ type AuthorizationValidationError struct {
 func (e *AuthorizationValidationError) Error() string {
 	return e.err
 }
+
+type ScrapeClass struct {
+	// Name of the scrape class, e.g. "istio-mtls".
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	Name string `json:"name"`
+
+	// Default designates a scrape class as the default class.
+	// +optional
+	Default *bool `json:"default,omitempty"`
```

Comment:
What happens if the user wants to create scrapeclasses, but not set a default?

Reply:
Nothing!
Scrape Class will be available for use but will not be added in any monitor resource